system-addon/content-src/styles/_icons.scss

```diff
@@ -39,6 +39,8 @@
   }
 
   &.icon-modal-delete {
+    flex-basis: auto;
```

Comment:
It looks like this is already the default so you can remove it. The line below alone fixes it for me.